### PR TITLE
Spearhead: fix import error

### PR DESCRIPTION
**Pull request by jffng**
On master, I'm getting an import error when trying to build Spearhead because of a typo here:

https://github.com/Automattic/themes/blob/49eead899692a426a2ad97caf0665f04c7f5bb8b/spearhead/assets/sass/style.scss#L17

This PR fixes it and rebuilds the stylesheets.

### CODE REVIEW

File: spearhead/assets/sass/style.scss [hunk 1]
```diff
@@ -14,7 +14,7 @@ Tags: one-column, flexible-header, accessibility-ready, custom-colors, custom-me
 */
 
 @import "../../../seedlet/assets/sass/abstracts/mixins";
-@import "../../../seedlet/assets/sass/base/accesibility";
+@import "../../../seedlet/assets/sass/base/accessibility";
```
Comment (scruffian) [inline spearhead/assets/sass/style.scss:17]: I think we should import these where they are actually needed

Comment (MaggieCabrera) [reply spearhead/assets/sass/style.scss:17]: That would be here, sorry for breaking things!

https://github.com/Automattic/themes/blob/49eead899692a426a2ad97caf0665f04c7f5bb8b/spearhead/assets/sass/_search.scss#L1

File: spearhead/assets/sass/_search.scss [hunk 2]
```diff
@@ -1,3 +1,6 @@
+// Import screen reader classes from Seedlet.
+@import "../../../seedlet/assets/sass/base/accessibility";
```
Comment (jffng) [inline spearhead/assets/sass/_search.scss:2]: @scruffian @MaggieCabrera is this a more logical place to import these styles?

File: spearhead/assets/sass/style.scss (hunk 1 above)
Comment (jffng) [reply spearhead/assets/sass/style.scss]: No worries, thanks!

**Pull request merged**